e2e/README.md

```diff
@@ -156,6 +156,18 @@ Functional tests are run by the `go test` command.
 go test ./e2e/ -timeout=20m -v -mod=vendor
 ```
 
+To run specific tests, you can specify options
+
+```console
+go test ./e2e/ --test-cephfs=false --test-rbd=false --upgrade-testing=true
+```
+
+To run e2e for specific tests with `make`, use
+
+```console
+make run-e2e E2E_ARGS="--test-cephfs=false --test-rbd=true --upgrade-testing=false"
+```
+
 You can also invoke functional tests with `make` command
```

Comment:
we have two commands in make to run e2e (run-e2e and func-test) I think we need to remove one.